tests/test_commands.py

```diff
@@ -1,5 +1,5 @@
 from django.test import TestCase
-from django.utils.six import StringIO
+from six import StringIO
```

Comment:
`django.utils.six` is part of the removed packages.

https://docs.djangoproject.com/en/3.0/releases/3.0/#removed-private-python-2-compatibility-apis